src/components/Navbar.js

```diff
@@ -0,0 +1,103 @@
+import { BellIcon, SettingsIcon, HamburgerIcon } from "@chakra-ui/icons"
+import { Avatar, AvatarBadge, AvatarGroup, Box, Button, Flex,
+    HStack, Text, IconButton, useDisclosure, Slide, Heading, Image } from "@chakra-ui/react"
+import { Link } from 'react-router-dom'
+import logo from "../logo.png"
+const Navbar = () => {
+
+    const { isOpen, onToggle } = useDisclosure()
+    const auth = JSON.parse(localStorage.getItem("auth"))
```

Comment:
Please use the auth hook to access the auth (user) data. Check how this is accessed in the other components:

```
const { user } = useAuth()
```

This way the logic for accessing the user (auth) data (taken from the local storage) lives in the useAuth hook only.

Reply:
Done